front-end/src/components/productView/ProductView.js

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { productService } from '../../services/productServices';
+
+const Container = styled.div`
+    display: flex;
+    justify-content: space-evenly;
+    padding: 2rem 0 2rem 0;
+    width: 80%;
+    min-height: 100vh;
+    background-color: whitesmoke;
+    margin: 0 auto 0 auto;
+    .letf__container {
+        p {
+            margin-bottom: 1rem;
+            color: #aaa;
+            font-size: 20px;
+            font-weight: 500;
+            
+        }
+        img {
+        width: 380px;
+        }
+    }
+    .right__container {
+        padding: 2rem;
+        margin-left: -10rem;
+        h2 {
+            font-weight: 600;
+            margin: 1rem 0 1rem 0;
+        }
+        p {
+            font-size: 20px;
+            font-weight: 300;
+            margin-bottom: 0.6rem;
+        }
+        .price {
+            margin-top: 5rem;
+            font-size: 25px;
+            font-weight:600;
+        }
+        div {
+            button{
+                margin: 2rem 1rem 0 0;
+                padding: 1.5rem;
+                background-color: white;
+                border: none;
+                border-radius: 0.5rem;
+                font-size: 20px;
+                cursor: pointer;
+                :hover {
+                    background-color: #bbb;
+                    color: white;
+                }
+            }
+        }
+    }
+    
+`;
+
+const ProductView = (props) => {
+    const redirect = useHistory();
+    const productId = props.location.pathname.split('/')[1];
```

Comment:
You can use useParams hook to get the id fron the url.
```suggestion
    const { id } = useParams();
```